src/type_checker/checker.rs

```diff
@@ -833,6 +835,19 @@ impl<B: Backend> TypeChecker<B> {
             None => (),
         };
 
+        // get the ignore_arg_types flag from the function info if it's a builtin
+        let ignore_arg_types = match self
```

Comment:
Seems to work for me but I am not sure if this is the correct way to check if a function is builtin.